documentation/InternalAPI.md

```diff
@@ -1,88 +0,0 @@
-Internal API Specification
---------------------------
-
-The following API is used by the frontend to get various information from the backend. The information is returned as JSON.
-It is not recommended for third parties to use this API, as it may change in the future.
-
-## GET /api/info
```

Comment:
Is this call actually removed?

Reply:
It's not removed, but it's not a public api either. I think there's no point in documenting the internal apis.

Reply:
Can we make this one "public"? I would say this is a useful one for administration purposes. I use it very often and it would be nice to have a place where others can see that it exists :)

Reply:
I see what you mean, is this ok? https://github.com/clarin-eric/switchboard-doc/pull/18/files#diff-c71fcc5b3f98b69327a6d8e2cdce9fca55f4d5354420c0410c82154761cabaa2R6-R8